cmd/gen-docs/main.go

```diff
@@ -0,0 +1,297 @@
+// Copyright 2019 the Kilo authors
```

Comment:
Can we make this 2021?

Reply:
yes, but don't they need to be the same dates on all files?

Reply:
The shell script doesn't care, as long as the header is there with any year